lib/matplotlib/font_manager.py

```diff
@@ -273,11 +273,26 @@ def get_fontconfig_fonts(fontext='ttf'):
 
     fontfiles = {}
     try:
-        warnings.warn('Matplotlib is building the font cache using fc-list. This may take a moment.')
         pipe = subprocess.Popen(['fc-list', '--format=%{file}\\n'],
                                 stdout=subprocess.PIPE,
                                 stderr=subprocess.PIPE)
-        output = pipe.communicate()[0]
+        # We emit a warning if the process has not completed after 5 seconds.
+        # We avoid showing it before since if this takes < 5 seconds, the user
+        # probably won't notice. However, this only works on Python 3 because
```

Comment:
subprocess32 doesn't have this? It is only an optional requirement, but that means that if people read our installation file (…), it'll work for 2.7